packages/plugin-ext/src/main/browser/comments/comment-thread-widget.tsx

```diff
@@ -50,6 +50,7 @@ export const COMMENT_TITLE: MenuPath = ['comment-title-menu'];
 export class CommentThreadWidget extends BaseWidget {
 
     protected readonly zoneWidget: MonacoEditorZoneWidget;
+    protected containerNodeRoot: Root;
```

Comment:
Minor: I'm wondering whether it makes sense to also declare this `readonly`, as it's only set in the constructor, right?

Reply:
I changed it to `readonly`.